pkgs/by-name/an/ansible-cmdb/package.nix

```diff
@@ -0,0 +1,54 @@
+{ lib
+, fetchFromGitHub
+, substituteAll
+, python3
+, testers
+, ansible-cmdb
+}:
+let
+  inherit (python3.pkgs)
+    mako
+    pyyaml
+    jsonxs
+    buildPythonApplication;
+
+  pname = "ansible-cmdb";
+  version = "1.31";
+in
+buildPythonApplication {
+  inherit pname version;
+  format = "setuptools";
```

Comment:
`format` is deprecated, [to my knowledge](https://github.com/NixOS/nixpkgs/issues/253154). Use `pyproject = false` instead

Reply:
Fixed.